### ISSUE TITLE
Support for .vtk, .vti in visualizer

### ISSUE THREAD

**Issue by rjsgml5698** [Kitware/paraviewweb#499]
Hello!

I implemented the paraviewweb visualizer as follows.

![image](https://user-images.githubusercontent.com/40594949/61930972-59fc5e00-afba-11e9-8b96-f662e4f2fb0a.png)

The .stl, .vtp files are implemented. However, .vtk and .vti files are not rendered.

.vtk, .vti files are not yet supported?

**Comment by jourdain** [comment]
Visualizer have access to the same reader as ParaView which means vtk and vti files can be loaded as well. 

To load them, you need to click on the "file" icon and click on each file that you want to load.

**Comment by rjsgml5698** [comment]
Thank you for your reply.

well...I already tried the following, but only the outline is showing. Can you explain if I'm missing something?

I would really appreciate it.

![image](https://user-images.githubusercontent.com/40594949/62195922-fa39f480-b3b7-11e9-9fdf-958a6a9162c1.png)

![image](https://user-images.githubusercontent.com/40594949/62195964-1473d280-b3b8-11e9-8cdc-c3e3d42a3043.png)

**Comment by jourdain** [comment]
That's expected as it is the default startup representation for Volume. But feel free to change it to what you want.

![62195964-1473d280-b3b8-11e9-8cdc-c3e3d42a3043](https://user-images.githubusercontent.com/294518/62227828-d8218180-b379-11e9-99d6-e0b9a359f2b8.png)

**Comment by rjsgml5698** [comment]
Through your answers I was able to solve the problem.

I really appreciate your reply.

**Comment by rjsgml5698** [comment]
Hello!

I have one more question.

I installed react with create-react-app.
And I saw that npm i paraviewweb was installed successfully.
![image](https://user-images.githubusercontent.com/40594949/62285641-f928c600-b490-11e9-9fec-5c1409b9c314.png)

However, the next picture and module do not seem to build properly.
![image](https://user-images.githubusercontent.com/40594949/62285575-d696ad00-b490-11e9-9959-16af7574c2bf.png)

When installing vtk.js in the create-react-app environment, I modified the webpack via npm i eject.

Should paraviewweb do the same thing as above? If so, can I refer to the link below?

http://kitware.github.io/paraviewweb/docs/setup.html

Thank you!
